src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,270 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: sa
+      changes:
+        - createSequence:
+            sequenceName: id_seq
+            incrementBy: 1
+            startValue: 1
+            
+  - changeSet:
+      id: 2
+      author: sa
+      changes:
+        - createSequence:
+            sequenceName: proc_type_id_seq
+            incrementBy: 1
+            startValue: 1          
+  - changeSet:
+      id: 3
+      author: sa
+      changes:
+        - createTable:
+            tableName: PROCESS_TYPE
+            columns:
+              - column:
+                  name: ID
```

Comment:
In Liquibase, you don't need to alter columns to add a primary key. Every ID column should have the following (after `autoIncrement`):
```yml
constraints:
  primaryKey: true
  nullable: false
```